docs/core/tools/custom-templates.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Create custom templates for dotnet new (.NET Core SDK 2.0 Preview 2) | Microsoft Docs
+description: 
+keywords: dotnet new, CLI, CLI command, .NET Core, template, templating
+author: guardrex
+ms.author: mairaw
+ms.date: 06/26/2017
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-cli
+ms.devlang: dotnet
+ms.assetid: c28eb59d-4d18-4aba-bcd1-88bd887abc08
+---
+
+# Create custom templates for dotnet new
+
+[!INCLUDE [core-preview-warning](~/includes/core-preview-warning.md)]
+
+The [.NET Core SDK](https://www.microsoft.com/net/download/core) provides many useful templates, but you can create your own custom templates for any type of .NET project, such as an app, service, tool, or class library. You can even create a template that outputs one or more independent files, such as a configuration file.
```

Comment:
wouldn't be better to say that The .NET Core SDK comes with many templates pre-installed for you to use with dotnet new?